packages/teleterm/src/ui/TopBar/NavigationMenu/NavigationMenu.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useRef } from 'react';
+import styled from 'styled-components';
+
+import Popover from 'design/Popover';
+import { MoreVert, OpenBox, Add } from 'design/Icon';
+import { Box, Text, Flex } from 'design';
+
+import { AccessRequestDocumentState } from 'teleterm/ui/services/workspacesService';
+
+import { useIdentity } from '../Identity/useIdentity';
+
+import { NavigationItem } from './NavigationItem';
+
+export function NavigationMenuContainer() {
+  const { activeRootCluster } = useIdentity();
+  return (
+    <NavigationMenu
+      clusterName={activeRootCluster?.name}
+      clusterUri={activeRootCluster?.uri}
+    />
+  );
+}
+
+const navigationItems: {
+  title: string;
+  Icon: JSX.Element;
+  state: AccessRequestDocumentState;
+}[] = [
+  {
+    title: 'New Access Request',
+    Icon: <Add fontSize={2} />,
+    state: 'creating',
+  },
+  {
+    title: 'Review Access Requests',
+    Icon: <OpenBox fontSize={2} />,
+    state: 'reviewing',
+  },
```

Comment:
Re: feature detection in Connect, do we plan to do that before the release of access requests?

Reply:
I believe that is to be a fast-follow